README.md

```diff
@@ -514,7 +514,7 @@ Thus when you create a TODO, it is almost always your name that is given.
      end
      ```
 
-* <a name="no-default-args"></a>Do not use default arguments. Use an options
+* <a name="no-default-args"></a>Do not use default positional arguments. Use named arguments or an options
```

Comment:
I think it's the inability to provide later values without providing the earlier ones. if we consider the `bad` example
```
def obliterate(things, gently = true, except = [], at = Time.now)
  ...
end
```
if someone wants to call `obliterate` with a custom `at` value, they must ALSO provide values for `gently` and `except`, even if they strictly want to use the default values. if we use the `good` signature, the caller will only specify the exact optional arguments it wants to provide.

i personally also agree that ordering is a problem for a high number of arguments // unintuitive arguments, but I also feel that discussion is tangential to optionality, which provides this sort of weird hard requirement